debian/pbspro-server.install

```diff
@@ -0,0 +1,308 @@
+etc/pbs_db_schema.sql /opt/pbs/etc
```

Comment:
Is there a way to use wildcards here? e.g. etc/* /opt/pbs/etc
That way when files get added they automatically get installed.